src/layouts/Layout.tsx

```diff
@@ -17,11 +17,12 @@ export default function Layout({ children }) {
 
   return (
     <>
-      <div className={`border-4 ${currentTheme || ""} border-skin-muted bg-skin-primary h-screen`}>
```

Comment:
Great! I actually had a note about this border as I was doing Auth this week. I support the removal!